tests/spread/plugins/v2/snaps/test-snapcraft-fake-ros2-package-core20/snap/snapcraft.yaml

```diff
@@ -0,0 +1,51 @@
+name: test-snapcraft-fake-ros2-package-core20
+base: core20
+version: '0.1'
+summary: Bare-bones ROS 2 dependency for snapcraft CLI test
+description: |
+  This snap contains a single unusable ROS 2 package. This snap is not meant to
```

Comment:
Unusable in the sense that it sense that it lacks practical utility?  The app should function, correct?

Reply:
That's right, the app function properly but in a way that's only useful to the test.